ntoskrnl/ke/i386/usercall.c

```diff
@@ -51,7 +51,7 @@ KiInitializeUserApc(IN PKEXCEPTION_FRAME ExceptionFrame,
                     IN PVOID SystemArgument1,
                     IN PVOID SystemArgument2)
 {
-    CONTEXT Context;
+    CONTEXT Context = { 0 };
```

Comment:
Does this really ensure that **all** the structure is zeroed out, and not just (say) the first member? (aka. is this behaviour the same on all compilers, GCC and MSVC?)

Reply:
Yes, a partial initializer causes the rest of the object to be zeroed.